src/plugins/commander/commands/member.js

```diff
@@ -77,9 +77,13 @@ class MemberCommand extends Command {
 
         const verifyUser = await this.getMastheadDiscord(userId);
 
-        if (verifyUser.value !== message.author.tag) {
+        if (message.author.discriminator?.length !== 4) {
```

Comment:
`discriminator` is currently `"0"` for new-style usernames, but [per docs](https://github.com/discord/discord-api-docs/blame/6e9622bec117934aba362bb8c6b0389ef4c3f794/docs/Change_Log.md#L25):

>  After migration of all users is complete, the `discriminator` field may be removed.